pkg/internal/common/logging/modlog/modlog.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package modlog
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/internal/common/logging/metadata"
+)
+
+//modLog is a moduled wrapper for api.Logger implementation
+//This adds functionality of module based levels on top of provider logger implementation
+type modLog struct {
+	logger log.Logger
+	module string
+}
+
+//Fatalf calls underlying logger.Fatal
```

Comment:
Why different method comments than defLog?

Reply:
defLog is implementation built on top of go standard logging,
modLog is built on top of logger injected through provider, mod log is wrapper added to control logger levels of logging provider injected.

Reply:
Ok. Hopefully "underlying" becomes more clear to the reader once the struct comment has more details.